src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,161 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> first;
+    private Node<T> last;
+    private int size = 0;
+
+    public MyLinkedList() {
+        first = new Node<>(null, null, null);
+        last = new Node<>(null, null, null);
+    }
```

Comment:
```suggestion
        first = new Node<>();
        last = new Node<>();
    }
```
just add default constructor to Node